cpp/ql/src/Security/CWE/CWE-327/BrokenCryptoAlgorithm.ql

```diff
@@ -16,9 +16,14 @@ abstract class InsecureCryptoSpec extends Locatable {
   abstract string description();
 }
 
+Function getAnInsecureFunction() {
+  result.getName().regexpMatch(algorithmBlacklistRegex()) and
+  exists(result.getACallToThisFunction())
```

Comment:
Out of interest, does the second line make much difference?

Reply:
I didn't investigate that, but I imagine that most functions are never called because most functions are declarations from libraries. At least that might be the case in a small project or a project using many libraries.